File: c11_svg/B_12_Emiliyan_Gospodinov.rb
def gime_a_red_circle x,y,r
  '<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'"
  stroke="black" stroke-width="4" fill="red" />'
end

def gime_a_yellow_circle x,y,r
  '<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'"
  stroke="black" stroke-width="4" fill="yellow" />'
end

def gime_a_green_circle x,y,r
  '<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'"
  stroke="black" stroke-width="4" fill="green" />'
end

def gime_a_rect w,h
 '<rect width="'+w.to_s+'" height="'+h.to_s+'" 
 style="fill:rgb(0,0,0);stroke-width:3;stroke:rgb(0,0,0)" />'
end

def gime_my_name x_,y_,name
  '<text x="'+x_.to_s+'" y="'+y_.to_s+'" fill="black">'+name.to_s+'</text>'
end
# lqvo,gore,obikolka
File.open("result.svg","w") do |f|
  f.write('<svg xmlns="http://www.w3.org/2000/svg">')
  f.write(gime_a_rect 40,100)
  f.write(gime_a_red_circle 22,20,15)
  f.write(gime_a_yellow_circle 22,50,15)
  f.write(gime_a_green_circle 22,80,15)
  f.write(gime_my_name 10,150,'Emiliyan Gospodinov')
  f.write('</svg>')
end

File: c21/B_12_Emiliyan_Gospodinov.js
var i = 0, count = 0, j = 0;
var array = [];
for (i = 0; i < 16; i++) {
    array[i] = i;   
}
for (i = 0; i < 4; i++) {
    for (j = 0; j < 4; j++){
        fill(0, 255, 17);
        rect(10 + i * 60, 10 + j* 60, 50, 50);
        fill(0, 0, 0);
        text(array[count],30 + i * 60, 30 + j * 60, 30, 50);
        count++;
    }
}
